guides/common/modules/proc_installing-satellite-ansible-modules-via-rpm.adoc

```diff
@@ -0,0 +1,35 @@
+[id="installing-satellite-ansible-modules-via-rpm_{context}"]
+== Installing the {Project} Ansible Modules from RPM
+
+Use this procedure to install the {Project} Ansible modules.
+
+ifeval::["{build}" == "satellite"]
+.Prerequisite
+
+* Ensure that the Ansible 2.9 or later repository is enabled.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# subscription-manager repos --enable rhel-7-server-ansible-2.9-rpms
+# {foreman-maintain} packages update ansible
+----
+endif::[]
+
+.Procedure
+
+ifeval::["{build}" == "satellite"]
+
+. Install the RPM using the following command:
```

Comment:
```suggestion
* Install the RPM using the following command:
```
Use a bulleted list instead of numbered list for a one-step procedure.

Reply:
Done